src/status_im2/contexts/profile/profiles/view.cljs

```diff
@@ -206,8 +208,14 @@
         :profile-picture     profile-picture
         :card-style          style/login-profile-card}]
       [standard-authentication/password-input
-       {:shell?           true
-        :default-password password}]]
+       {:shell?              true
+        :on-press-biometrics (fn []
```

Comment:
is this correct? Have you handled the case where biometrics fails -> password input shows, then you press biometric authenticate again and let it fail -> password input shows then you press biometric authenticate again ... (to infinity :) )

Reply:
Not sure I understand the flow 😄

Reply:
I might be wrong let me test out your branch 😀